packages/sitecore-jss/src/graphql-request-client.ts

```diff
@@ -4,16 +4,29 @@ import { DocumentNode } from 'graphql';
 import debuggers, { Debugger } from './debug';
 import TimeoutPromise from './utils/timeout-promise';
 
+/**
+ * Options for configuring a GraphQL request.
+ */
+interface RequestOptions {
+  [key: string]: unknown;
```

Comment:
Let's avoid using index signature since we shouldn't really be able to pass any values here. After you remove index signature you will see that GraphQLEditingService doesn't pass "sc_layoutKind" as it is expected